tests/unit/s2n_server_new_session_ticket_test.c

```diff
@@ -430,9 +428,11 @@ int main(int argc, char **argv)
          *# expanded (32 octets):  4e cd 0e b6 ec 3b 4d 87 f5 d6 02 8f 92 2c
          *# a4 c5 85 1a 27 7f d4 13 11 c9 e6 2d 2c 94 92 e1 c4 f3
          **/
+        /* clang-format off */
         S2N_BLOB_FROM_HEX(expected_session_secret,
-        "4e cd 0e b6 ec 3b 4d 87 f5 d6 02 8f 92 2c \
+                       "4e cd 0e b6 ec 3b 4d 87 f5 d6 02 8f 92 2c \
             a4 c5 85 1a 27 7f d4 13 11 c9 e6 2d 2c 94 92 e1 c4 f3");
+        /* clang-format on */
```

Comment:
Based on nst_message below, it looks like the alignment here is supposed to match the comment? Like 4e in the comment should line up with 4e in the blob. This one didn't match before though so you could leave it as-is.

Reply:
I think I will leave it as is. At the very best its unclear.